### PR TITLE
Fixing namespace for image samples

### PR DESCRIPTION
**Pull request by abgoswam**
Towards #3205 

- fixing the Image samples (which were previously updated via #3165)

### CODE REVIEW

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/DnnFeaturizeImage.cs [hunk 1]
```diff
@@ -15,7 +16,7 @@ public static void Example()
             // Downloading a few images, and an images.tsv file, which contains a list of the files from the dotnet/machinelearning/test/data/images/.
             // If you inspect the fileSystem, after running this line, an "images" folder will be created, containing 4 images, and a .tsv file
             // enumerating the images. 
-            var imagesDataFile = SamplesUtils.DatasetUtils.DownloadImages();
+            var imagesDataFile = Microsoft.ML.SamplesUtils.DatasetUtils.DownloadImages();
```
Comment (rogancarr) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/DnnFeaturizeImage.cs]: >Microsoft.ML. [](start = 33, length = 13)

Do you need this?

**Comment by rogancarr** [comment]
>             // 0.394427955

Hmmmmmmmmmmm

---
Refers to: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/DnnFeaturizeImage.cs:62 in e178288. [](commit_id = e178288f6adf91d7eb1d754bb122526335ccefe2, deletion_comment = False)

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/DnnFeaturizeImage.cs (hunk 1 above)
Comment (abgoswam) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/DnnFeaturizeImage.cs]: added to `ConvertToGrayScale` in iteration 2  (else its compile time error)

---
In reply to: [272409998](https://github.com/dotnet/machinelearning/pull/3208#discussion_r272409998) [](ancestors = 272409998)

**Comment by abgoswam** [comment]
>             // 0.394427955

ResNet18  ?  :) 

---
In reply to: [480105797](https://github.com/dotnet/machinelearning/pull/3208#issuecomment-480105797) [](ancestors = 480105797)

---
Refers to: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ImageAnalytics/DnnFeaturizeImage.cs:62 in e178288. [](commit_id = e178288f6adf91d7eb1d754bb122526335ccefe2, deletion_comment = False)

**Pull request merged**